<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSelect</class>
 <widget class="QGroupBox" name="SoundSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound selection</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="labelVoicepacks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>VoicePacks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCategories">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Categories:</string>
   </property>
  </widget>
  <widget class="QListView" name="listFiles">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listCategories">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listVoicepacks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>271</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelFiles">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Voice files:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOkay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="filterCategories">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filter categories</string>
   </property>
   <property name="statusTip">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="filterFiles">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filter voice files</string>
   </property>
   <property name="statusTip">
    <string>Filter</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlaySound">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Play Sound</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStopSound">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Stop Sound</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
